state/test/trie/test_proof.py

```diff
@@ -258,11 +258,12 @@ def test_proof_prefix_only_prefix_nodes():
     for k, v in key_vals.items():
         node_trie.update(k.encode(), rlp_encode([v]))
 
-    proof_nodes = node_trie.generate_state_proof_for_key_prfx(prefix.encode())
+    proof_nodes, val = node_trie.generate_state_proof_for_keys_with_prefix(prefix.encode(), get_value=True)
```

Comment:
Do we have a test where `get_value=False`?

Reply:
Added